lib/client-assets.php

```diff
@@ -111,6 +111,10 @@ function register_tinymce_scripts() {
 function gutenberg_override_script( $handle, $src, $deps = array(), $ver = false, $in_footer = false ) {
 	wp_deregister_script( $handle );
 	wp_register_script( $handle, $src, $deps, $ver, $in_footer );
+
+	if ( function_exists( 'wp_set_script_translations' ) ) {
```

Comment:
Do we need `function_exists` at this point? Assuming we can change the minimum required version for the plugin to 5.0+, which seems a reasonable expectation.